templates/register.twig

```diff
@@ -0,0 +1,204 @@
+                            </p>
+                        </template>
+                        <template v-if="slide === 2">
+                            <p>
+                                Please click and read <a target="_blank" href="https://gametactic.eu/tos"> terms of service </a> fully.
+                                When you're done, please agree them bellow.
+                                <v-checkbox v-model="tos" @click="cannotContinue = false" :disabled="tos" label="I Accept Terms of Service" />
+                            </p>
+                        </template>
+                        <template v-if="slide === 3">
+                            <v-form v-model="usernameValid">
+                                <p> Please select username. It will be same, even you login via different service. </p>
+                                <v-text-field v-model="username" :rules="nameRules" label="Username" :counter="32"
+                                    placeholder="{{ username }}" @input="onProfileFields()" required />
+                                <v-select v-model="region" :items="regions" label="Region" item-value="val" item-text="key" @change="onProfileFields()"/>
+                            </v-form>
+                        </template>
+                        <template v-if="slide === 4">
+                            <h2>Thank you {{ '{{ username }}' }}!</h2>
+                            <br>
+                            <p>
+                                You are now ready to go. Just press "continue" and you will be redirected
+                                back where you left off.
+                            </p>
+                        </template>
+                    </v-form>
+                </template>
+                <template v-if="'{{ error }}' === 'registered'">
+                    <p> You have already registered! </p>
+                </template>
+            </v-card-text>
+            <v-divider style="margin-top: 15px" />
+            <v-card-actions v-if="'{{ error }}' === ''">
+                <v-spacer />
+                <v-btn color="link" @click="previousSlide()" :disabled="slide <= 2 || slide === 4">Go Back</v-btn>
+                <v-btn color="primary" @click="nextSlide()" :disabled="cannotContinue">Continue</v-btn>
+            </v-card-actions>
+        </div>
+        `,
+        data() {
+            return {
+                slide: 1,
+                cannotContinue: false,
+                tos: false,
+                username: '',
+                usernameValid: false,
+                region: '',
+                regions: [
+                    { val: 'eu', key: 'Europe' },
+                    { val: 'asia', key: 'Asia' },
+                    { val: 'ru', key: 'Russia' },
+                    { val: 'na', key: 'North America' },
+                ],
+                nameRules: [
+                    v => !!v || 'Username is required',
```

Comment:
Try and use the same casing for the same word throughout the app. Is it Username or username, Terms of service or Terms of Service or terms of service?

Consistency is key !:)

Reply:
Depends where it is. If its first word of sentence, it is Username, otherwise username